packages/desktop/lib/routers/utils/getRouterForAppContext.ts

```diff
@@ -0,0 +1,21 @@
+import { get } from 'svelte/store'
+
+import { AppContext } from '@core/app/enums'
+import { IRouter } from '@core/router/interfaces'
+import { dashboardRouter, onboardingRouter, settingsRouter } from '@core/router/routers'
+import { loginRouter } from '@core/router/subrouters'
+
+export function getRouterForAppContext(context: AppContext): IRouter {
+    switch (context) {
+        case AppContext.Login:
+            return get(loginRouter)
+        case AppContext.Dashboard:
+            return get(dashboardRouter)
```

Comment:
Very very very minor thing, but could you do this alphabetically? It will help a lot when we need to compare `getSubroutersForAppContext` with this function.